libsrc/telestrat/keyboard.s

```diff
@@ -0,0 +1,14 @@
+; jede jede@oric.org 2017-01-22
+
+    .export         _key
+				
+    .include        "zeropage.inc"		
+    .include        "telestrat.inc"
+
+; char key(void);
+		
+.proc _key
+    BRK_TELEMON XRDW0 ; read keyboard
```

Comment:
As far as I understand this is the same as a `read()` from fd 0. In that case please remove it. cc65 C libraries are not supposed to duplicate functionality already present through standard APIs. Just check out the other targets. Most of them have some nice straight-forward ROM entry point for "just" get a char from the keyboard, but non of them has some `key()` C function.